src/components/MoneyTemporaryForRefactorRequestConfirmationList.js

```diff
@@ -624,6 +627,233 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
         );
     }, [isReadOnly, iouType, selectedParticipants.length, confirm, bankAccountRoute, iouCurrencyCode, policyID, splitOrRequestOptions, formError, styles.ph1, styles.mb2]);
 
+    // All fields that a request can have
```

Comment:
...instead of this one here.

Reply:
The intention was to:
- make the code easier to grasp in general
- explain the "classified" part of the local variable name

Reply:
Yeah, let's put the more explanatory comment here please